Accordion/Responsive_Accordion_01/code/style.css

```diff
@@ -0,0 +1,127 @@
+html {
+    width: 80%;
+    min-width: 150px;
+    height: 100%;
+    margin: 0 auto;
+    overflow-y: scroll;
+    background-color: #FFF;
+    font-family:  sans-serif;
+    color: rgba(48, 69, 92, 0.8);
+  }  
+  .accordion-bral {
```

Comment:
Indentation here seems weird. For guidelines you can refer to [this](https://google.github.io/styleguide/htmlcssguide.html)